src/pal/src/thread/context.cpp

```diff
@@ -481,7 +481,7 @@ void CONTEXTFromNativeContext(const native_context_t *native, LPCONTEXT lpContex
     
     if ((contextFlags & CONTEXT_FLOATING_POINT) == CONTEXT_FLOATING_POINT)
     {
-#ifdef _AMD64_
+#if 0//def _AMD64_
```

Comment:
This code is generally a quick hack to run on Alpine, as I said. I don't
know a good way to detect musl or Alpine to do this.

On Fri, 12 Feb 2016, 11:24 Kamil Rytarowski <notifications@github.com>
wrote:

> In src/pal/src/thread/context.cpp
> <https://github.com/dotnet/coreclr/pull/2262#discussion_r52730507>:
>
> > @@ -481,7 +481,7 @@ void CONTEXTFromNativeContext(const native_context_t *native, LPCONTEXT lpContex
> >
> >      if ((contextFlags & CONTEXT_FLOATING_POINT) == CONTEXT_FLOATING_POINT)
> >      {
> > -#ifdef _AMD64_
> > +#if 0//def _AMD64_
>
> why for everybody?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dotnet/coreclr/pull/2262/files#r52730507>.
>